src/components/ContextMenu.vue

```diff
@@ -12,22 +12,30 @@
 export const mixin = {
   methods: {
     showContextMenu(event) {
-      this.$store.dispatch("showContextMenu", {x: event.x, y: event.y});
-      let items = [];
+      let pos = { x: event.x, y: event.y };
+      var items = [];
```

Comment:
Why did this become var?

Reply:
Oh was because I was trying to fix the bug where nothing was showing up, wasn't sure items were working and thought it might have been something to do with the scope. It wasn't, and can be `let` again now.